src/MIDebugEngine/AD7.Impl/AD7Engine.cs

```diff
@@ -572,13 +572,20 @@ public int Continue(IDebugThread2 pThread)
             // VS Code currently isn't providing a thread Id in certain cases. Work around this by handling null values.
             AD7Thread thread = pThread as AD7Thread;
 
-            if (_pollThread.IsPollThread())
+            try
             {
-                _debuggedProcess.Continue(thread?.GetDebuggedThread());
+                if (_pollThread.IsPollThread())
+                {
+                    _debuggedProcess.Continue(thread?.GetDebuggedThread());
+                }
+                else
+                {
+                    _pollThread.RunOperation(() => _debuggedProcess.Continue(thread?.GetDebuggedThread()));
```

Comment:
Does pullthread.runoperation carry the exception back here if the exception is thrown from the other thread?